src/main/java/core/basesyntax/services/WriterService.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.services;
+
+public interface WriterService {
+    void writeReportToFile(String report);
```

Comment:
```suggestion
    void writeToFile(String filePath, String text);
```
Why report? Can't this interface write any strings?